dashboard/app/models/census/ap_school_code.rb

```diff
@@ -55,7 +56,7 @@ def self.seed_from_s3
 
   def self.seed
     if CDO.stub_school_data
-      seed_from_csv("test/fixtures/census/ap_school_codes.csv")
+      seed_from_csv("test/fixtures/census/ap_school_codes.csv", 2016)
```

Comment:
Could we borrow the convention from the state course offerings seeding logic where the school year is in the filename and picked up dynamically?

Reply:
Sure, I can make that change. Any thoughts on how to test this change more broadly?

Reply:
Ended up doing what you suggested here. I tested the impact of moving the existing `ap_school_code.csv` file to a place like `"ap_school_codes/#{school_year}-#{school_year + 1}.csv"` by:

1. copying down existing `ap_school_codes` table from production
2. doing complete seeding of `ap_school_codes` using files in new location
3. checking whether any rows with school_year = 2016 (ie, existing data) had a modified `updated_at` (they did not).